indico/htdocs/js/indico/jquery/ajaxform.js

```diff
@@ -0,0 +1,254 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* global showFormErrors:false, initForms:false, confirmPrompt:false, updateHtml:false */
+/* global getDropzoneFiles:false, setDropzoneFiles:false */
+
+(function(global) {
+    'use strict';
+
+    global.inlineAjaxForm = function inlineAjaxForm(options) {
+        options = $.extend(true, {
+            // the element used to trigger events etc. may also be a selector.
+            // when using it, the element should not be inside any element
+            // that will be updated by this function as events would
+            // not bubble up properly.  in many cases using the button that
+            // triggers the loading of the form may be the most suitable
+            // element unless it is being replaced with the form, in which case
+            // its parent element may be a good candidate.
+            context: null,
```

Comment:
It's not really a container but just the element we trigger events on - usually it's the button that triggered loading or in case of a already-available form the form itself